Rakefile

```diff
@@ -157,6 +157,6 @@ end
 #
 def replace_version_number(new_version_number)
   text = File.read(podspec_path)
-  text.gsub!(/(s.version( )*= ")#{spec_version}(")/, "\\1#{new_version_number}\\3")
+  text.gsub!(/"#{spec_version}"/, "\"#{new_version_number}\"")
```

Comment:
Understood that this will match all instances of the current version and replace with new version, and this will help handle potential future changes to the podspec (e.g. new field added that contains the version). However, I wonder if this is maybe too broad. While unlikely, this could eventually match against text in "s.license" or "s.platform".

It terms of finding the balance between:
a) future changes that are missed by current regex
b) unintended changes to fields

I would give higher priority to 'b', as a point of caution. Even just copy-pasting the original regex and then adjusting the new instance to update "s.source.tag" seems fine. I think the tiny bit of duplicated code that would result is perfectly reasonable.

Reply:
changed it, was trying to do a fancy regex before but I'm not very good at it. Simplified it now, one regex per case (s.version and s.source.tag).